md/deploy-bundle.md

```diff
@@ -23,10 +23,89 @@ It is useful to update the configuration, locally or from a remote computer.
 
 1. Download the deploy.zip file from the [Bonitasoft download page](http://www.bonitasoft.com/downloads-v2) for the Community edition 
 or from the [Customer Portal](https://customer.bonitasoft.com/download/request) for Subscription editions.
-2. Unzip the `deploy.zip` into a temporary folder.
-3. Install Bonita BPM in your Application Server, by merging the appropriate folder (e.g. `Wildfly-10.1.0.Final` or `Tomcat-7.0.67`) from `BonitaBPMSubscription-x.y.z-deploy` folder
-into the existing root folder of your application server.
-4. Follow the [Tomcat bundle steps](tomcat-bundle.md#configuration) or [WildFly bundle steps](wildfly-bundle.md#configuration)
+2. Unzip the `deploy.zip` into a temporary folder (DEPLOY_ZIP_HOME).
+4. Follow the [Tomcat bundle steps](#tomcat-installation) or [WildFly bundle steps](#wildfly-installation)
+
+<a id="tomcat-installation" />
+
+## Tomcat installation
+### Install Bonita BPM Platform in Tomcat
+
+Copy all files and directories from DEPLOY_ZIP_HOME/Tomcat-7.0.76/server to your Tomcat root directory (TOMCAT_HOME).
+Warning: Some configurations files from the deploy zip will overwrite some default tomcat configuration files. Proceed
+with care in a tomcat where some applications are already installed.
+
+### Configure data sources
+
+Configuration of data source for Tomcat is in two parts: because Tomcat doesn't support JTA natively,
+one data source will be configured in the Bitronix configuration file and the other data source will be configured
+in the standard Tomcat context configuration file.
+
+Non-transactional data source
+1. Open the file TOMCAT_HOME/conf/Catalina/localhost/bonita.xml
+2. Remove or comment out the lines regarding the h2 database.
+3. Uncomment the line matching your RDBMS.
+4. Update following attributes value:
+    - username: your RDBMS user name.
+    - password: your RDBMS password.
+    - url: the URL, including the RDBMS server address, RDBMS server port and database (schema) name.
+5. Repeat operations 2. to 4. for business data data source
+
+JTA data source (managed by Bitronix)
+1. Open the file TOMCAT_HOME/conf/bitronix-resources.properties
+2. Remove or comment out the lines regarding the h2 database.
+3. Uncomment the line matching your RDBMS.
+4. Update the value for each of the following settings:
+    - resource.ds1.driverProperties.user: your RDBMS user name.
+    - resource.ds1.driverProperties.password: your RDBMS password.
+    - resource.ds1.driverProperties.serverName: the address (IP or hostname) of your RDBMS	server.
+    - resource.ds1.driverProperties.portNumber: put the port of your RDBMS server.
+    - resource.ds1.driverProperties.databaseName: put the database name.
+5. Repeat operations 2. to 4. for business data data source (resources.ds2)
```

Comment:
This is not so simple: 
* **Postgres** requires to set separately serverName, portNumber, databaseName
* Other RDBMS require to set URL directly